phoenix-landing-page/Phoenix_files/_hero.scss

```diff
@@ -0,0 +1,82 @@
+@import url(http://fonts.googleapis.com/css?family=Lato:300,400,700);
+
+body, h1, h2, h3, h4, h5, h6 {
+  font-family: 'Lato', sans-serif !important;
+}
+
+.header {
+  @include radial_gradient(circle at 50% 50%, #F67938 29%, #EF4734 120%);
```

Comment:
how do you feel about moving `#F67938` and `#EF4734` to variables? (same for other colors in this file)